### PR TITLE
WT-5994 WiredTiger does not currently support relaxed isolation levels, turn off format testing

### PR DESCRIPTION
**Pull request by keithbostic**

### CODE REVIEW

**Comment by keithbostic** [comment]
> is all of this sufficiently isolated or readily identifiable so that when restrictions aren't so rigid we can undo it? If not, do we want to go through and label/comment lines that restrict so we know to remove them?

I think (but haven't tested), that removing the first couple of lines of the `config_transaction()` code should be sufficient, the rest of that function is correct for a world where we can test more aggressively.

File: test/format/config.c [hunk 1]
```diff
@@ -766,75 +766,95 @@ config_pct(void)
...
-        }
+    if (g.c_prepare && config_is_perm("ops.prepare")) {
+        if (g.c_logging && config_is_perm("logging"))
+            testutil_die(EINVAL, "prepare is incompatible with logging");
+        if (!g.c_txn_timestamps && config_is_perm("transaction.timestamps"))
+            testutil_die(EINVAL, "prepare requires transaction timestamps");
+        if (g.c_isolation_flag != ISOLATION_SNAPSHOT && config_is_perm("transaction.isolation"))
+            testutil_die(EINVAL, "prepare requires snapshot isolation");
+        if (g.c_txn_freq != 100 && config_is_perm("transaction.frequency"))
+            testutil_die(EINVAL, "prepare requires transaction frequency set to 100");
```
Comment (sueloverso) [inline test/format/config.c:794]: Where did this requirement come from?

Comment (keithbostic) [reply test/format/config.c:794]: Not sure what you're asking?

Comment (sueloverso) [reply test/format/config.c:794]: I don't see the frequency=100 in the "before" code.

Comment (keithbostic) [reply test/format/config.c:794]: Oh, I added some stuff that was missing as part of review. Basically, timestamps require everything to be inside of a transction, atm.

**Pull request merged**